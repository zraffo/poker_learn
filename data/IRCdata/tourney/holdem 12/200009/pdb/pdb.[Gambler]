[Gambler] 968210378 20 13 f   -     -     -        10000    0    0 
[Gambler] 968210562 20 12 f   -     -     -        10000    0    0 
[Gambler] 968210664 20 11 f   -     -     -        10000    0    0 
[Gambler] 968210723 20 10 f   -     -     -        10000    0    0 
[Gambler] 968210807 20  9 c   c     f     -        10000  950    0 
[Gambler] 968210918 20  8 f   -     -     -         9050    0    0 
[Gambler] 968210985 20  7 f   -     -     -         9050    0    0 
[Gambler] 968211113 20  6 cc  kf    -     -         9050 1100    0 
[Gambler] 968211198 19  5 f   -     -     -         7950    0    0 
[Gambler] 968211296 19  4 cc  kf    -     -         7950  800    0 
[Gambler] 968211444 17  2 f   -     -     -         7150    0    0 
[Gambler] 968211515 16  2 BccA -     -     -         7150 7150    0 5c 6c
[Gambler] 969512421 10  6 f   -     -     -        10000    0    0 
[Gambler] 969512563 10  5 f   -     -     -        10000    0    0 
[Gambler] 969512655 10  4 rc  b     bA    -        10000 10000  100 Js Ad
[Gambler] 969512750 10  3 f   -     -     -          100    0    0 
[Gambler] 969513007 10  2 BA  -     -     -          100  100    0 Kd 5h
