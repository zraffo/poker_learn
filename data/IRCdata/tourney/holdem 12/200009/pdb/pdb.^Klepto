^Klepto   968562958 10  3 f   -     -     -       100000    0    0 
^Klepto   968563040 10  2 Bk  b     bc    k       100000 23250 47250 8s 4c
^Klepto   968563100 10  1 Bc  kc    kc    kc      124000 99500    0 Js Jd
^Klepto   968563186 10 10 c   k     k     f        24500 1000    0 
^Klepto   968563244 10  9 c   k     k     k        23500 1000    0 Qh Jc
^Klepto   968563318 10  8 c   f     -     -        22500 1000    0 
^Klepto   968563390 10  7 f   -     -     -        21500    0    0 
^Klepto   968563429 10  6 f   -     -     -        21500    0    0 
^Klepto   968563475 10  5 ccA -     -     -        21500 21500    0 Qs Ad
^Klepto   968564652 12  5 f   -     -     -       250000    0    0 
^Klepto   968564705 12  4 f   -     -     -       250000    0    0 
^Klepto   968564792 12  3 f   -     -     -       250000    0    0 
^Klepto   968564866 12  2 Bc  k     k     kf      250000 9000    0 
^Klepto   968564923 12  1 Bc  kf    -     -       241000 4000    0 
^Klepto   968565021 12 12 c   k     k     b       237000 20000 24000 5c Qc
^Klepto   968565079 12 11 r   bcA   -     -       241000 241000 562000 5s 5d
^Klepto   968565122 12 10 c   r     -     -       562000 316000 66000 3d Ad
^Klepto   968565186 10  7 f   -     -     -       312000    0    0 
^Klepto   968565226 10  7 f   -     -     -       312000    0    0 
^Klepto   968565275  8  6 f   -     -     -       312000    0    0 
^Klepto   968565317  8  5 f   -     -     -       312000    0    0 
^Klepto   968565354  8  4 cc  cA    -     -       312000 312000    0 Qc 2c
^Klepto   968566321  8  8 c   f     -     -       100000  500    0 
^Klepto   968566436  8  7 c   f     -     -        99500 15500    0 
^Klepto   968566496  8  6 f   -     -     -        84000    0    0 
^Klepto   968566560  8  5 f   -     -     -        84000    0    0 
^Klepto   968566663  8  4 f   -     -     -        84000    0    0 
^Klepto   968566704  8  3 f   -     -     -        84000    0    0 
^Klepto   968566774  8  2 Bf  -     -     -        84000 2000    0 
^Klepto   968566851  8  1 Bf  -     -     -        82000 2000    0 
^Klepto   968566922  8  8 c   k     k     f        80000 4000    0 
^Klepto   968566974  8  7 f   -     -     -        76000    0    0 
^Klepto   968566999  7  5 f   -     -     -        76000    0    0 
^Klepto   968567063  7  4 c   c     k     f        76000 24000    0 
^Klepto   968567132  7  3 rA  -     -     -        52000 52000    0 3c Jd
^Klepto   969843214 18 10 f   -     -     -         2000    0    0 
^Klepto   969843318 17  8 cc  k     k     kc        2000  350    0 Th 7d
^Klepto   969843489 17  7 c   kc    kf    -         1650  110    0 
^Klepto   969843639 15  6 f   -     -     -         1540    0    0 
^Klepto   969843775 15  5 c   r     cA    -         1540 1540    0 8d 6s
^Klepto   969846643 17 10 r   bcA   -     -        10000 10000    0 Kc Kd
^Klepto   970021444 15  2 Bk  kf    -     -        10000   50    0 
^Klepto   970021545 15  1 Bc  kf    -     -         9950  175    0 
^Klepto   970021601 15 15 f   -     -     -         9775    0    0 
^Klepto   970021676 15 14 f   -     -     -         9775    0    0 
^Klepto   970021781 15 13 f   -     -     -         9775    0    0 
^Klepto   970021904 15 12 f   -     -     -         9775    0    0 
^Klepto   970022024 15 11 c   k     k     k         9775  100    0 As 8d
^Klepto   970022103 15 10 c   k     c     f         9675  800    0 
^Klepto   970022219 15  9 c   k     cc    f         8875 2500    0 
^Klepto   970022316 15  8 f   -     -     -         6375    0    0 
^Klepto   970022424 14  6 f   -     -     -         6375    0    0 
^Klepto   970022516 14  5 f   -     -     -         6375    0    0 
^Klepto   970022578 14  4 f   -     -     -         6375    0    0 
^Klepto   970022615 13  3 c   c     cA    -         6375 6375    0 Kd 5d
^Klepto   970024836 19 12 cc  kf    -     -         5000  725    0 
^Klepto   970024968 18 11 c   f     -     -         4275   50    0 
^Klepto   970025046 16  9 f   -     -     -         4225    0    0 
^Klepto   970025130 16  8 f   -     -     -         4225    0    0 
^Klepto   970025216 16  7 f   -     -     -         4225    0    0 
^Klepto   970025325 15  6 f   -     -     -         4225    0    0 
^Klepto   970025499 15  5 f   -     -     -         4225    0    0 
^Klepto   970025548 14  4 f   -     -     -         4225    0    0 
^Klepto   970025611 14  3 f   -     -     -         4225    0    0 
^Klepto   970025655 14  2 Bk  k     kf    -         4225  100    0 
^Klepto   970025780 13  1 Bf  -     -     -         4125  100    0 
^Klepto   970025839 12 12 f   -     -     -         4025    0    0 
^Klepto   970025910 12 11 f   -     -     -         4025    0    0 
^Klepto   970025997 12 10 f   -     -     -         4025    0    0 
^Klepto   970026116 11  8 f   -     -     -         4025    0    0 
^Klepto   970026170 11  7 rc  k     cA    -         4025 4025    0 Ah Kh
