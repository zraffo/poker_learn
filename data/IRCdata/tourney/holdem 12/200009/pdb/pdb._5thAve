_5thAve   968891893 16  7 f   -     -     -        10000    0    0 
_5thAve   968891992 16  6 f   -     -     -        10000    0    0 
_5thAve   968892090 15  5 c   cf    -     -        10000  550    0 
_5thAve   968892154 14  4 c   kc    kc    k         9450 4250    0 Qs Kc
_5thAve   968892210 14  3 c   bc    k     b         5200 3625 4950 
_5thAve   968892266 14  2 Bc  kcA   -     -         6525 6525 13125 Qh 4h
_5thAve   968892306 14  1 Bc  k     kf    -        13125  350    0 
_5thAve   968892403 13 13 f   -     -     -        12775    0    0 
_5thAve   968892465 13 12 c   f     -     -        12775  750    0 
_5thAve   968892535 13 11 f   -     -     -        12025    0    0 
_5thAve   968892616 12  9 f   -     -     -        12025    0    0 
_5thAve   968892663 12  9 rc  bA    -     -        12025 12025 2525 Qd Qc
_5thAve   968892701 12  8 f   -     -     -         2525    0    0 
_5thAve   968892800 12  7 c   b     bA    -         2525 2525 5650 Ks 2s
_5thAve   968892836 12  6 c   c     k     cA        5650 5650 11900 7d 7h
_5thAve   968892894 12  5 cc  c     kcA   -        11900 11900 15400 6d As
_5thAve   968892966 12  4 f   -     -     -        15400    0    0 
_5thAve   968893006 12  3 cc  kc    k     b        15400 14400 22500 9h 9d
_5thAve   968893061 11  2 Bf  -     -     -        23500  200    0 
_5thAve   968893089 11  1 Bc  k     k     kf       23300  200    0 
_5thAve   968893170 11 11 f   -     -     -        23100    0    0 
_5thAve   968893223 11 10 c   c     -     -        23100 7375 18150 9h Ks
_5thAve   968893278 10  8 f   -     -     -        33875    0    0 
_5thAve   968893313 10  7 c   kf    -     -        33875  400    0 
_5thAve   968893356 10  6 cc  f     -     -        33475 2400    0 
_5thAve   968893395 10  5 c   c     r     -        31075 27600 46525 Qd 7d
_5thAve   968893459  9  4 c   k     f     -        50000  400    0 
_5thAve   968893503  9  3 c   kc    k     k        49600 2000    0 5d 6d
_5thAve   968893568  9  2 Br  -     -     -        47600 2200 3600 
_5thAve   968893604  9  1 Bc  kf    -     -        49000  400    0 
_5thAve   968893664  9  9 c   r     b     -        48600 28000 49275 4s Ac
_5thAve   968893708  8  7 c   k     c     c        69875 4900 11400 Ac Kc
_5thAve   968893785  7  6 c   c     -     -        76375 9225    0 Qc 8h
_5thAve   968893819  7  5 c   k     k     c        67150 3600    0 5h 5c
_5thAve   968893868  7  4 f   -     -     -        63550    0    0 
_5thAve   968893900  7  3 f   -     -     -        63550    0    0 
_5thAve   968893996  7  2 Bf  -     -     -        63550  800    0 
_5thAve   968894036  7  1 Bf  -     -     -        62750  400    0 
_5thAve   968894053  7  7 c   b     -     -        62350 4000 6400 
_5thAve   968894112  7  6 c   k     k     k        64750 1600    0 5h 6c
_5thAve   968894171  7  5 f   -     -     -        63150    0    0 
_5thAve   968894190  7  4 c   k     b     -        63150 6400 9600 
_5thAve   968894272  7  3 f   -     -     -        66350    0    0 
_5thAve   968894345  7  2 Bk  f     -     -        66350 1600    0 
_5thAve   968894372  7  1 Bf  -     -     -        64750  800    0 
_5thAve   968894390  7  7 f   -     -     -        63950    0    0 
_5thAve   968894403  7  6 c   c     f     -        63950 8000    0 
_5thAve   968894435  7  5 c   rA    -     -        55950 55950 78400 Js Tc
_5thAve   968894461  6  3 c   cc    -     -        78400 21275    0 Kd 5d
_5thAve   968894487  6  2 Bc  -     -     -        57125 2350 5500 Ac Kh
_5thAve   968894506  5  1 Bc  bc    -     -        60275 20100 40200 Ks 8h
_5thAve   968894570  4  4 f   -     -     -        80375    0    0 
_5thAve   968894600  4  4 r   -     -     -        80375 11200 16000 
_5thAve   968894624  4  3 c   k     k     k        85175 3200 8000 Kd 9s
_5thAve   968894707  4  2 Bf  -     -     -        89975 3200    0 
_5thAve   968894713  4  1 Bf  -     -     -        86775 1600    0 
_5thAve   968894730  4  4 c   f     -     -        85175 3200    0 
_5thAve   968894750  4  3 cc  b     -     -        81975 35200 48000 
_5thAve   968894774  4  2 Bf  -     -     -        94775 3200    0 
_5thAve   968894828  4  1 Bf  -     -     -        91575 1600    0 
_5thAve   968894834  4  4 r   -     -     -        89975 11200 16000 
_5thAve   968894841  4  3 c   k     b     -        94775 11200 16000 
_5thAve   968894956  4  2 Bk  k     b     k        99575 11200 24000 Qs 2s
_5thAve   968894993  4  1 Bc  -     -     -       112375 12650    0 9d Tc
_5thAve   968895003  3  3 c   k     b     -        99725 22400 32000 
_5thAve   968895031  3  2 Bf  -     -     -       109325 6400    0 
_5thAve   968895036  3  1 Bc  b     -     -       102925 19200 25600 
_5thAve   968895048  3  3 r   k     b     -       109325 70400 117375 6c 6s
_5thAve   968895067  2  1 B   -     -     -       156300 6400 6400 
_5thAve   968895068  2  1 Bf  -     -     -       156300 3200    0 
_5thAve   968895073  2  2 B   -     -     -       153100 6400 9600 
_5thAve   968895074  2  1 Bc  -     -     -       156300 3700    0 3h 6s
_5thAve   968895076  2  2 B   -     -     -       152600 6400 9600 
_5thAve   968895077  2  1 Bc  -     -     -       155800 4200    0 6s 4s
_5thAve   968895082  2  2 B   -     -     -       151600 6400 9600 
_5thAve   968895083  2  1 Bc  -     -     -       154800 5200    0 Js Ts
_5thAve   968895088  2  2 B   -     -     -       149600 6400 9600 
_5thAve   968895089  2  1 Bc  k     k     k       152800 6400    0 2s Ac
_5thAve   968895105  2  2 B   -     -     -       146400 6400 9600 
_5thAve   968895106  2  1 Bc  k     kf    -       149600 6400    0 
_5thAve   968895124  2  2 Bf  -     -     -       143200 6400    0 
_5thAve   968895127  2  1 Bcf -     -     -       136800 6400    0 
_5thAve   968895131  2  2 Bc  c     -     -       130400 29600 59200 6s Ks
_5thAve   968895388 19  9 f   -     -     -         5000    0    0 
_5thAve   968895501 19  8 f   -     -     -         5000    0    0 
_5thAve   968895645 19  7 f   -     -     -         5000    0    0 
_5thAve   968895755 19  6 f   -     -     -         5000    0    0 
_5thAve   968895845 17  5 f   -     -     -         5000    0    0 
_5thAve   968895971 15  3 f   -     -     -         5000    0    0 
_5thAve   968896034 14  2 f   -     -     -         5000    0    0 
_5thAve   968896138 14  2 Bf  -     -     -         5000  100    0 
_5thAve   968896215 14  1 Bf  -     -     -         4900   50    0 
_5thAve   968896302 14 14 f   -     -     -         4850    0    0 
_5thAve   968896379 13 12 f   -     -     -         4850    0    0 
_5thAve   968896471 13 11 f   -     -     -         4850    0    0 
_5thAve   968896514 13 10 f   -     -     -         4850    0    0 
_5thAve   968896561 13  9 f   -     -     -         4850    0    0 
_5thAve   968896655 13  8 f   -     -     -         4850    0    0 
_5thAve   968896745 13  7 rA  -     -     -         4850 4850    0 Kd Ac
_5thAve   969145056 23 14 f   -     -     -        10000    0    0 
_5thAve   969145207 21 12 cc  kc    kc    bA       10000 10000 21050 Jc Td
_5thAve   969145345 20 11 c   b     b     bc       21050 10000 21175 Qs 8h
_5thAve   969145471 19 10 cf  -     -     -        32225  100    0 
_5thAve   969145536 19  9 f   -     -     -        32125    0    0 
_5thAve   969145588 18  8 cc  k     kf    -        32125  900    0 
_5thAve   969145672 18  7 cc  kc    k     cf       31225 10000    0 
_5thAve   969145774 17  5 c   b     b     -        21225 3350 5200 
_5thAve   969145887 17  4 f   -     -     -        23075    0    0 
_5thAve   969145959 16  3 f   -     -     -        23075    0    0 
_5thAve   969146179 16  2 Bc  kf    -     -        23075 2275    0 
_5thAve   969146276 16  1 Bc  kf    -     -        20800 1100    0 
_5thAve   969146363 16 16 c   c     -     -        19700 11300    0 9h Jc
_5thAve   969146446 16 15 f   -     -     -         8400    0    0 
_5thAve   969146519 16 14 f   -     -     -         8400    0    0 
_5thAve   969146586 16 13 f   -     -     -         8400    0    0 
_5thAve   969146659 15 11 f   -     -     -         8400    0    0 
_5thAve   969146713 15 10 c   f     -     -         8400 1400    0 
_5thAve   969146781 15  9 f   -     -     -         7000    0    0 
_5thAve   969146833 14  7 f   -     -     -         7000    0    0 
_5thAve   969146879 12  5 f   -     -     -         7000    0    0 
_5thAve   969146919 11  3 f   -     -     -         7000    0    0 
_5thAve   969146983 11  2 Bc  kf    -     -         7000 3600    0 
_5thAve   969147068 11  1 Bf  -     -     -         3400  400    0 
_5thAve   969147098 11 11 c   k     cA    -         3000 3000    0 7s Qh
_5thAve   969495333 18  5 f   -     -     -         7500    0    0 
_5thAve   969495502 18  4 cc  k     rcA   -         7500 7500 15900 5c 4d
_5thAve   969495626 17  2 c   kc    kc    kf       15900 1550    0 
_5thAve   969495677 17  2 Bk  kc    k     b        14350 10707 10820 5h As
_5thAve   969495746 17  1 Bc  k     k     k        14463  211    0 Kh 7h
_5thAve   969495844 17 17 c   f     -     -        14252  100    0 
_5thAve   969495943 17 16 c   f     -     -        14152  300    0 
_5thAve   969496020 17 15 c   f     -     -        13852  100    0 
_5thAve   969496124 16 13 f   -     -     -        13752    0    0 
_5thAve   969496210 16 12 f   -     -     -        13752    0    0 
_5thAve   969496275 16 11 f   -     -     -        13752    0    0 
_5thAve   969496339 16 10 f   -     -     -        13752    0    0 
_5thAve   969496417 12  6 c   kf    -     -        13752  200    0 
_5thAve   969496534 12  6 f   -     -     -        13552    0    0 
_5thAve   969496553 12  5 f   -     -     -        13552    0    0 
_5thAve   969496595 12  4 c   kf    -     -        13552  400    0 
_5thAve   969496654 12  3 cc  k     kc    kf       13152 1800    0 
_5thAve   969496729 12  2 Bf  -     -     -        11352  400    0 
_5thAve   969496755 11  1 Bf  -     -     -        10952  200    0 
_5thAve   969496824 11 11 cc  c     cA    -        10752 10752 17145 Ad 4d
_5thAve   969496898 10 10 f   -     -     -        17145    0    0 
_5thAve   969496944  8  7 cc  f     -     -        17145 3200    0 
_5thAve   969496997  8  7 f   -     -     -        13945    0    0 
_5thAve   969497028  8  6 c   c     c     -        13945 3388    0 8c Jc
_5thAve   969497063  8  5 cA  -     -     -        10557 10557    0 Qd Td
