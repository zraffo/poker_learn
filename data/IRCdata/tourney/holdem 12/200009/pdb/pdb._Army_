_Army_    969185481  7  4 f   -     -     -        10000    0    0 
_Army_    969185542  7  3 f   -     -     -        10000    0    0 
_Army_    969185555  7  2 Bf  -     -     -        10000   50    0 
_Army_    969185600  7  1 Bf  -     -     -         9950   25    0 
_Army_    969185697  7  7 f   -     -     -         9925    0    0 
_Army_    969185714  7  6 c   b     k     k         9925  250    0 4h Qs
_Army_    969185750  7  5 cc  kf    -     -         9675  225    0 
_Army_    969185774  7  4 f   -     -     -         9450    0    0 
_Army_    969185797  7  3 cc  kf    -     -         9450  225    0 
_Army_    969185837  7  2 Bk  c     f     -         9225  150    0 
_Army_    969185892  7  1 Bf  -     -     -         9075   50    0 
_Army_    969185919  7  7 c   b     -     -         9025  350  500 
_Army_    969185944  7  6 c   f     -     -         9175  100    0 
_Army_    969185988  7  5 r   b     -     -         9075 1100 1500 
_Army_    969186028  7  4 cf  -     -     -         9475  100    0 
_Army_    969186079  7  3 r   b     -     -         9375 1100 1500 
_Army_    969186108  7  2 Bk  c     k     k         9775  400  900 9d 8c
_Army_    969186162  7  1 Bc  kf    -     -        10275  100    0 
_Army_    969186184  7  7 c   f     -     -        10175  100    0 
_Army_    969186234  6  5 f   -     -     -        10075    0    0 
_Army_    969186262  6  4 c   b     -     -        10075  900 1400 
_Army_    969186297  6  3 f   -     -     -        10575    0    0 
_Army_    969186362  5  1 B   -     -     -        10575  200  200 
_Army_    969186382  5  1 Bf  -     -     -        10575  100    0 
_Army_    969186394  5  5 f   -     -     -        10475    0    0 
_Army_    969186421  5  4 c   f     -     -        10475  200    0 
_Army_    969186438  5  3 f   -     -     -        10275    0    0 
_Army_    969186445  5  2 Bk  k     kf    -        10275  200    0 
_Army_    969186484  5  1 Brf -     -     -        10075  800    0 
_Army_    969186501  5  5 c   f     -     -         9275  200    0 
_Army_    969186518  5  4 c   k     f     -         9075  200    0 
_Army_    969186562  5  3 r   -     -     -         8875  700 1000 
_Army_    969186569  5  2 Bf  -     -     -         9175  200    0 
_Army_    969186595  5  1 Bc  b     -     -         8975  600  800 
_Army_    969186620  5  5 f   -     -     -         9175    0    0 
_Army_    969186641  5  4 f   -     -     -         9175    0    0 
_Army_    969186654  5  3 c   k     k     k         9175  400    0 Kd 9h
_Army_    969186686  5  2 Br  bA    -     -         8775 8775    0 Jd Jc
