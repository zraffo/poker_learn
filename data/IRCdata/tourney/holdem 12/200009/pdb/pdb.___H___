___H___   970274224 10  9 f   -     -     -        10000    0    0 
___H___   970274267 10  8 f   -     -     -        10000    0    0 
___H___   970274306 10  7 f   -     -     -        10000    0    0 
___H___   970274370 10  6 cf  -     -     -        10000   50    0 
___H___   970274420 10  5 f   -     -     -         9950    0    0 
___H___   970274474 10  4 f   -     -     -         9950    0    0 
___H___   970274537  9  2 Bf  -     -     -         9950  100    0 
___H___   970274589  9  1 Bf  -     -     -         9850   50    0 
___H___   970274640  9  9 f   -     -     -         9800    0    0 
___H___   970274685  8  7 c   c     k     b         9800 5550 5000 8h 8s
___H___   970274737  8  6 cc  k     k     f         9250  550    0 
___H___   970274799  8  5 cccA -     -     -         8700 8700 28050 Th Ah
___H___   970274831  6  3 cc  f     -     -        28050 2000    0 
___H___   970274895  4  2 Bk  b     -     -        26050 3200 3600 
___H___   970274916  4  1 Bc  kf    -     -        26450  200    0 
___H___   970274939  4  4 c   f     -     -        26250  200    0 
___H___   970274965  4  3 rc  bcA   -     -        26050 26050    0 5d 5h
___H___   970306173  6  3 f   -     -     -        15000    0    0 
___H___   970306361  6  2 Bk  kf    -     -        15000  150    0 
___H___   970306375  6  1 Bf  -     -     -        14850   75    0 
___H___   970306390  6  6 f   -     -     -        14775    0    0 
___H___   970306403  6  5 f   -     -     -        14775    0    0 
___H___   970306424  6  4 f   -     -     -        14775    0    0 
___H___   970306446  6  3 f   -     -     -        14775    0    0 
___H___   970306452  6  2 Bf  -     -     -        14775  150    0 
___H___   970306476  6  1 Bf  -     -     -        14625  150    0 
___H___   970306533  6  6 f   -     -     -        14475    0    0 
___H___   970306538  6  5 f   -     -     -        14475    0    0 
___H___   970306543  6  4 f   -     -     -        14475    0    0 
___H___   970306549  6  3 f   -     -     -        14475    0    0 
___H___   970306557  6  2 Bk  kf    -     -        14475  300    0 
___H___   970306565  6  1 Bf  -     -     -        14175  150    0 
___H___   970306571  6  6 f   -     -     -        14025    0    0 
___H___   970306589  6  5 f   -     -     -        14025    0    0 
___H___   970306596  6  4 f   -     -     -        14025    0    0 
___H___   970306600  6  3 f   -     -     -        14025    0    0 
___H___   970306609  6  2 Bk  k     kf    -        14025  300    0 
___H___   970306622  6  1 Bf  -     -     -        13725  150    0 
___H___   970306625  6  6 f   -     -     -        13575    0    0 
___H___   970306631  6  5 f   -     -     -        13575    0    0 
___H___   970306638  6  4 f   -     -     -        13575    0    0 
___H___   970306645  6  3 f   -     -     -        13575    0    0 
___H___   970306656  6  2 Bk  kf    -     -        13575  300    0 
___H___   970306663  6  1 Bf  -     -     -        13275  150    0 
___H___   970306671  6  6 f   -     -     -        13125    0    0 
___H___   970306672  6  5 f   -     -     -        13125    0    0 
___H___   970306680  6  4 f   -     -     -        13125    0    0 
___H___   970306689  6  3 f   -     -     -        13125    0    0 
___H___   970306710  6  2 Bk  k     kf    -        13125  300    0 
___H___   970306722  6  1 Bf  -     -     -        12825  150    0 
___H___   970306725  6  6 f   -     -     -        12675    0    0 
___H___   970306726  6  5 f   -     -     -        12675    0    0 
___H___   970306733  6  4 f   -     -     -        12675    0    0 
___H___   970306740  6  3 f   -     -     -        12675    0    0 
___H___   970306747  6  2 Bk  kf    -     -        12675  300    0 
___H___   970306757  6  1 Bf  -     -     -        12375  150    0 
___H___   970306764  6  6 f   -     -     -        12225    0    0 
___H___   970306768  6  5 f   -     -     -        12225    0    0 
___H___   970306776  6  4 f   -     -     -        12225    0    0 
___H___   970306787  6  3 f   -     -     -        12225    0    0 
___H___   970306795  6  2 Bk  kf    -     -        12225  600    0 
___H___   970306802  6  1 Bf  -     -     -        11625  300    0 
___H___   970306808  6  6 f   -     -     -        11325    0    0 
___H___   970306809  6  5 f   -     -     -        11325    0    0 
___H___   970306817  6  4 f   -     -     -        11325    0    0 
___H___   970306823  6  3 f   -     -     -        11325    0    0 
___H___   970306841  6  2 Bk  k     k     kf       11325  600    0 
___H___   970306861  6  1 Bf  -     -     -        10725  300    0 
___H___   970306880  6  6 f   -     -     -        10425    0    0 
___H___   970306896  6  5 f   -     -     -        10425    0    0 
___H___   970306900  6  4 f   -     -     -        10425    0    0 
___H___   970306906  6  3 f   -     -     -        10425    0    0 
___H___   970306917  6  2 Bf  -     -     -        10425  600    0 
___H___   970306925  6  1 Bf  -     -     -         9825  300    0 
___H___   970306948  6  6 f   -     -     -         9525    0    0 
___H___   970306952  6  5 f   -     -     -         9525    0    0 
___H___   970306985  6  4 f   -     -     -         9525    0    0 
___H___   970307004  6  3 f   -     -     -         9525    0    0 
___H___   970307026  5  2 Bk  f     -     -         9525  600    0 
___H___   970307038  5  1 Bf  -     -     -         8925  300    0 
___H___   970307048  5  5 f   -     -     -         8625    0    0 
___H___   970307053  5  4 f   -     -     -         8625    0    0 
___H___   970307058  5  3 f   -     -     -         8625    0    0 
___H___   970307071  5  2 Bf  -     -     -         8625 1200    0 
___H___   970307079  5  1 Bf  -     -     -         7425  600    0 
___H___   970307086  5  5 f   -     -     -         6825    0    0 
___H___   970307087  5  4 f   -     -     -         6825    0    0 
___H___   970307092  5  3 f   -     -     -         6825    0    0 
___H___   970307095  5  2 Bk  kf    -     -         6825 1200    0 
___H___   970307104  5  1 Bf  -     -     -         5625  600    0 
___H___   970307111  5  5 f   -     -     -         5025    0    0 
___H___   970307113  5  4 f   -     -     -         5025    0    0 
___H___   970307121  5  3 f   -     -     -         5025    0    0 
___H___   970307126  5  2 Bk  kf    -     -         5025 1200    0 
___H___   970307133  5  1 Bf  -     -     -         3825  600    0 
___H___   970307139  5  5 f   -     -     -         3225    0    0 
___H___   970307141  5  4 f   -     -     -         3225    0    0 
___H___   970307149  5  3 f   -     -     -         3225    0    0 
___H___   970307162  5  2 Bk  kf    -     -         3225 1200    0 
___H___   970307172  5  1 Bf  -     -     -         2025  600    0 
___H___   970307175  5  5 f   -     -     -         1425    0    0 
___H___   970307180  5  4 f   -     -     -         1425    0    0 
___H___   970307187  5  3 f   -     -     -         1425    0    0 
___H___   970307195  4  2 Bk  k     k     k         1425 1200 3600 3h 3c
___H___   970307212  4  1 Bf  -     -     -         3825  600    0 
___H___   970307223  4  4 f   -     -     -         3225    0    0 
___H___   970307224  4  3 f   -     -     -         3225    0    0 
___H___   970307233  4  2 Bk  kf    -     -         3225 1200    0 
___H___   970307240  4  1 Bf  -     -     -         2025  600    0 
___H___   970307247  4  4 f   -     -     -         1425    0    0 
___H___   970307249  4  3 f   -     -     -         1425    0    0 
___H___   970307263  4  2 Bk  kf    -     -         1425 1200    0 
___H___   970307272  4  1 BA  -     -     -          225  225    0 4c Kd
