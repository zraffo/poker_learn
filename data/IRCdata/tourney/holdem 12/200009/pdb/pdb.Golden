Golden    968958721 14  1 Bc  k     kf    -        10000   50    0 
Golden    968958886 14 14 f   -     -     -         9950    0    0 
Golden    968958947 12 11 f   -     -     -         9950    0    0 
Golden    968959031 12 10 c   c     f     -         9950 1225    0 
Golden    968959139 11  8 f   -     -     -         8725    0    0 
Golden    968959201 11  7 f   -     -     -         8725    0    0 
Golden    968959276 11  6 f   -     -     -         8725    0    0 
Golden    968959410 11  5 f   -     -     -         8725    0    0 
Golden    968959468 11  4 f   -     -     -         8725    0    0 
Golden    968959538 11  3 f   -     -     -         8725    0    0 
Golden    968959601 10  1 Bc  kf    -     -         8725  800    0 
Golden    968959669  9  1 Bf  -     -     -         7925  100    0 
Golden    968959821  9  9 c   b     -     -         7825  900 1400 
Golden    968959890  9  8 f   -     -     -         8325    0    0 
Golden    968959965  8  6 c   f     -     -         8325 1200    0 
Golden    968960026  8  6 r   -     -     -         7125 1400 2000 
Golden    968960054  8  5 f   -     -     -         7725    0    0 
Golden    968960094  8  4 r   b     -     -         7725 7000 11200 
Golden    968960131  8  3 f   -     -     -        11925    0    0 
Golden    968960163  8  2 Br  bA    -     -        11925 11925 14025 
Golden    968960202  8  1 Br  b     b     -        14025 10800 18875 5c 5h
Golden    968960238  7  7 f   -     -     -        22100    0    0 
Golden    968960401  7  7 c   b     -     -        22100 2800 4000 
Golden    968960426  7  6 rc  -     -     -        23300 9625 20450 2h 2d
Golden    968960463  6  4 f   -     -     -        34125    0    0 
Golden    968960509  6  3 f   -     -     -        34125    0    0 
Golden    968960526  6  2 Bk  k     k     k        34125  800    0 5s 8h
Golden    968960565  6  1 Bf  -     -     -        33325  400    0 
Golden    968960596  5  5 f   -     -     -        32925    0    0 
Golden    968960624  5  5 c   k     k     k        32925 2800    0 2c Kc
Golden    968960655  5  4 r   b     -     -        30125 11200 16800 
Golden    968960704  5  3 r   -     -     -        35725 2800 4000 
Golden    968960718  5  2 Bf  -     -     -        36925  800    0 
Golden    968960744  5  1 Bf  -     -     -        36125  400    0 
Golden    968960774  5  5 r   bA    -     -        35725 35725 45325 
Golden    968960809  5  4 f   -     -     -        45325    0    0 
Golden    968960823  5  3 f   -     -     -        45325    0    0 
Golden    968960842  5  2 Bk  k     b     -        45325 6400 9600 
Golden    968960894  5  1 Bf  -     -     -        48525  800    0 
Golden    968960997  5  5 f   -     -     -        47725    0    0 
Golden    968961020  5  4 f   -     -     -        47725    0    0 
Golden    968961039  5  3 f   -     -     -        47725    0    0 
Golden    968961067  5  2 Bc  kf    -     -        47725 5600    0 
Golden    968961117  5  1 Br  -     -     -        42125 4800 6400 
Golden    968961130  5  5 f   -     -     -        43725    0    0 
Golden    968961170  5  4 r   -     -     -        43725 11200 16000 
Golden    968961185  5  3 r   -     -     -        48525 11200 16000 
Golden    968961200  5  2 B   -     -     -        53325 3200 4800 
Golden    968961214  5  1 Bf  -     -     -        54925 1600    0 
Golden    968961247  4  4 r   k     b     -        53325 28800 38400 
Golden    968961280  4  4 f   -     -     -        62925    0    0 
Golden    968961332  3  3 r   -     -     -        62925 11200 16000 
Golden    968961351  3  2 Bf  -     -     -        67725 3200    0 
Golden    968961361  3  1 Bc  b     -     -        64525 9600 12800 
Golden    968961375  3  3 f   -     -     -        67725    0    0 
Golden    968961381  3  2 Bf  -     -     -        67725 3200    0 
Golden    968961386  3  1 Brc bA    -     -        64525 64525 9925 Kh 6h
Golden    968961401  3  3 f   -     -     -         9925    0    0 
Golden    968961412  3  2 Bk  bA    -     -         9925 9925 14725 
Golden    968961430  3  1 Br  bA    -     -        14725 14725    0 Kh 7h
